packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx

```diff
@@ -130,6 +151,17 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
           {displayFiat(props.order.inputQuantity)} {props.order.inputCurrency}
         </Value>
       </Row>
+      <Row>
+        <Title>
+          <FormattedMessage
+            id='modals.simplebuy.confirm.payment'
+            defaultMessage='Payment Method'
+          />
+        </Title>
+        <Value>
+          {props.order.paymentMethodId || getPaymentMethod(props.order)}
```

Comment:
This means card method id (which is a random string) is that what you want?

Reply:
@plondon it was there already, so I keep it, let me know if we do not need it I'll remove